addons/l10n_ec/models/account_move.py

```diff
@@ -134,6 +134,8 @@ class AccountMove(models.Model):
         string="Payment Method (SRI)",
     )
 
+    _get_l10n_ec_identification_type = _get_l10n_ec_ats_identification_type #For backward compatibility, remove in master
```

Comment:
@pepetreshere This breaks the module installation. Please place it after the method definition (not doing myself to avoid conflicts)